applications/GeoMechanicsApplication/custom_constitutive/coulomb_with_tension_cut_off_impl.cpp

```diff
@@ -158,4 +191,20 @@ void CoulombWithTensionCutOffImpl::load(Serializer& rSerializer)
     rSerializer.load("TensionCutOff", mTensionCutOff);
 }
 
+double CoulombWithTensionCutOffImpl::CalculateEquivalentPlasticStrain(const Vector& rSigmaTau,
+                                                                      CoulombYieldSurface::CoulombAveragingType AveragingType,
+                                                                      double lambda) const
+{
+    Vector dGdsigma = mCoulombYieldSurface.DerivativeOfFlowFunction(rSigmaTau, AveragingType);
+    double g1       = (dGdsigma[0] + dGdsigma[1]) * 0.5;
+    double g3       = (dGdsigma[0] - dGdsigma[1]) * 0.5;
+
+    double m_mean       = (g1 + g3) / 3.0;
+    double m_deviatoric = std::sqrt(std::pow(g1 - m_mean, 2) + std::pow(g3 - m_mean, 2));
+    double alpha        = std::sqrt(2.0 / 3.0) * m_deviatoric;
+    double delta_kappa  = -alpha * lambda;
+
+    return delta_kappa;
```

Comment:
Here we can use const auto in most places and we could return -alpha*lambda instead of caching it first
```suggestion
    const auto dGdsigma = mCoulombYieldSurface.DerivativeOfFlowFunction(rSigmaTau, AveragingType);
    const auto g1       = (dGdsigma[0] + dGdsigma[1]) * 0.5;
    const auto g3       = (dGdsigma[0] - dGdsigma[1]) * 0.5;

    const auto m_mean       = (g1 + g3) / 3.0;
    const auto m_deviatoric = std::sqrt(std::pow(g1 - m_mean, 2) + std::pow(g3 - m_mean, 2));
    const auto alpha        = std::sqrt(2.0 / 3.0) * m_deviatoric;
   return -alpha * lambda;
```

Reply:
Similar to the function of lambda, here also wanted to make it clear that we are solving for delta_kappa. So, I tried to write delta-kappa explicitly